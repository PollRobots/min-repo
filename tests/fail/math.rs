use llvm_cov_358_min_repo::fizzbuz;

pub fn main() {
    let _ = fizzbuz! {
        1,
        1,
        3,
        4,
        5,
        6,
        7,
        8,
        7,
        8,
        11,
        buzz,
        fizz,
        fizz_buzz,
        15
    };
}
