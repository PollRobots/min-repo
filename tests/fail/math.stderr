error: Expecting 2
 --> tests/fail/math.rs:6:9
  |
6 |         1,
  |         ^

error: Expecting fizz
 --> tests/fail/math.rs:7:9
  |
7 |         3,
  |         ^

error: Expecting buzz
 --> tests/fail/math.rs:9:9
  |
9 |         5,
  |         ^

error: Expecting fizz
  --> tests/fail/math.rs:10:9
   |
10 |         6,
   |         ^

error: Expecting fizz
  --> tests/fail/math.rs:13:9
   |
13 |         7,
   |         ^

error: Expecting buzz
  --> tests/fail/math.rs:14:9
   |
14 |         8,
   |         ^

error: Expecting fizz
  --> tests/fail/math.rs:16:9
   |
16 |         buzz,
   |         ^^^^

error: Expecting 13
  --> tests/fail/math.rs:17:9
   |
17 |         fizz,
   |         ^^^^

error: Expecting 14
  --> tests/fail/math.rs:18:9
   |
18 |         fizz_buzz,
   |         ^^^^^^^^^

error: Expecting fizz_buzz
  --> tests/fail/math.rs:19:9
   |
19 |         15
   |         ^^
